examples/js/local.js

```diff
@@ -33,13 +33,25 @@ const uploadFiles = () => {
 
```

Comment:
Ah, i just still see some changes to this file so I asked, maybe this was unintentional?

Reply:
It's intentional, do you want me to remove the changes since focus has since shifted to archive.js?

Reply:
What is the purpose of the changes, does it do the same thing for the local.html demo? That makes sense if so!

Reply:
The changes expand existing drag and drop functionality in local.html/local.js to include support JSON file.

Reply:
OK, great, let's keep it!

Reply:
Okay, many thanks!